src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -82,11 +122,19 @@ public void addCompletionTime() {
         facts.add(new Fact(NAME_COMPLETION_TIME, TimeUtils.dateToString(countCompletionTime())));
     }
 
-    private long countCompletionTime() {
-        long duration = run.getDuration() == 0L
-                ? System.currentTimeMillis() - run.getStartTimeInMillis()
-                : run.getDuration();
-        return run.getStartTimeInMillis() + duration;
+    private long getRunDuration()
+    {
+        long duration;
+        if (run.getDuration() == 0L)
+            duration = System.currentTimeMillis() - run.getStartTimeInMillis();
+        else 
+            duration = run.getDuration();
+        return duration;
+    }
+    
+    private long countCompletionTime() 
+    {        
```

Comment:
wrong code formatting

Reply:
can you please clarify what is wrong here?

Reply:
the bracket `{` should in line with the method

Reply:
@casz , thank!